frame/conviction-voting/src/tests.rs

```diff
@@ -57,7 +57,10 @@ impl Contains<Call> for BaseFilter {
 
 parameter_types! {
 	pub BlockWeights: frame_system::limits::BlockWeights =
-		frame_system::limits::BlockWeights::simple_max(1_000_000);
+		frame_system::limits::BlockWeights::simple_max(frame_support::weights::Weight::new()
+			.set_computation(1024)
+			.set_bandwidth(1024)
```

Comment:
nit: if this is supposed to mimic a builder patter, I think the `set` prefix is pretty redundant

Reply:
Otherwise it conflict with the field name